packages/test/test-end-to-end-tests/src/test/summarizerWithLocalChanges.spec.ts

```diff
@@ -701,4 +714,55 @@ describeNoCompat("Summarizer with local changes", (getTestObjectProvider) => {
 			);
 		},
 	);
+
+	itExpects(
+		"Errors thrown from mixinSummaryHandler are tagged as DataProcessingError",
+		[
+			{
+				eventName: "fluid:telemetry:Summarizer:Running:Summarize_cancel",
+				clientType: "noninteractive/summarizer",
+				error: "Mixed-in summary handler threw!",
+				errorType: "dataProcessingError",
+			},
+		],
+		async () => {
+			// The "summarize" event is only emitted when this setting is enabled
+			settings["Fluid.Summarizer.UseDynamicRetries"] = true;
+
+			const container = await createContainer(provider, false /* disableSummary */);
+			await waitForContainerConnection(container);
+
+			const rootDataObject = await requestFluidObject<RootTestDataObject>(container, "/");
+			const containerRuntime = rootDataObject.containerRuntime as ContainerRuntime;
+
+			try {
```

Comment:
You can try something like `await assert.rejects(` as well. Total non-blocker feel free to ignore

Reply:
This `try` block is to ensure `containerRuntime.dispose` is called.  Nothing in here rejects or throws.